### PR TITLE
worker host network action

### PR DESCRIPTION
**Pull request by njuettner**
add a new action fo host network worker verification

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: pkg/action/cl001/ac007/explainer.go [hunk 1]
```diff
@@ -5,5 +5,13 @@ import (
 )
 
 func (e *Explainer) explain(ctx context.Context) (string, error) {
-	return "", nil
+	s := `
+Check if the number of host network pods on tenant cluster worker nodes matches the number we expect from k8scloudconfig.
+
+	* Fetch all Tenant Cluster nodes and take the the first worker node by label.
+	* Compare the current pods with host network set with the expected amount of pods on worker node.
+	* See also https://github.com/giantswarm/k8scloudconfig/blob/529491d591e039da1ffde03fef070101c8d4a95c/files/conf/setup-kubelet-environment#L21.
```
Comment (xh3b4sd) [inline pkg/action/cl001/ac007/explainer.go:13]: This is not the right line number pal. 😛

Comment (xh3b4sd) [reply pkg/action/cl001/ac007/explainer.go:13]: Also, https://github.com/giantswarm/k8scloudconfig/blob/529491d591e039da1ffde03fef070101c8d4a95c/files/conf/setup-kubelet-environment#L26 says `6` but the tests here say `5`. What is it now? That has to got fixed somehow.

Comment (njuettner) [reply pkg/action/cl001/ac007/explainer.go]: Forgot to update the explainer and the executor, it's 6 pods as the k8scloudconfig says.

Comment (njuettner) [reply pkg/action/cl001/ac007/explainer.go]: Updated the line, thanks 👍

**Pull request merged**